tasks/eth/sony-001-holocene-system-config-upgrade/VALIDATION.md

```diff
@@ -0,0 +1,26 @@
+# Validation
+
+This document can be used to validate the state diff resulting from the execution of the upgrade
+transaction.
+
+For each contract listed in the state diff, please verify that no contracts or state changes shown in the Tenderly diff are missing from this document. Additionally, please verify that for each contract:
+
+- The following state changes (and none others) are made to that contract. This validates that no unexpected state changes occur.
+- All addresses (in section headers and storage values) match the provided name, using the Etherscan and Superchain Registry links provided. This validates the bytecode deployed at the addresses contains the correct logic.
+- All key values match the semantic meaning provided, which can be validated using the storage layout links provided.
+
+## State Changes
+
+### `0x7A8Ed66B319911A0F3E7288BDdAB30d9c0C875c3` (`SystemConfigProxy`)
+
+- **Key**: `0x0000000000000000000000000000000000000000000000000000000000000068`
+  **Before**: `0x0000000000000000000000000000000000000000000000000000000001c9c380`
+  **After**: `0x0000000000000000000000000000000000177fef000026080000000001c9c380`
```

Comment:
Non-blocking comment: get someone to validate that these values are correct @sebastianst @geoknee

Reply:
Prior to holocene you could query a L2 predeploy to get the scalar values:

```
 % cast call 0x420000000000000000000000000000000000000F 'baseFeeScalar()(uint256)' --rpc-url https://rpc.soneium.org
9736
```

```
% cast call 0x420000000000000000000000000000000000000F 'blobBaseFeeScalar()(uint256)' --rpc-url https://rpc.soneium.org
1540079 [1.54e6]
```